docs/build/index.md

```diff
@@ -25,24 +104,55 @@ We recommend using [Homebrew](https://brew.sh) to install all dependencies. Once
 brew install numworks/tap/epsilon-sdk
 ```
 
+<br>
+
 ### Debian or Ubuntu
 
 Most of the required tools are available as apt packages:
 
 ```
-apt-get install build-essential git imagemagick libx11-dev libxext-dev libfreetype6-dev libpng-dev libjpeg-dev pkg-config
+apt-get install build-essential git imagemagick libx11-dev libxext-dev libfreetype6-dev libpng-dev libjpeg-dev pkg-config apt-get install gcc-arm-none-eabi binutils-arm-none-eabi
 ```
 
-You'll also need to install the latest version of GCC and make it available in your $PATH:
+<br>
 
-1. Download the [GCC toolchain distributed by ARM](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads). You should obtain a `gcc-arm-none-eabi-x-linux.tar.bz2` file.
-2. Decompress that file with `tar xvfj gcc-arm-none-eabi-*-linux.tar.bz2`
-3. Add the resulting folder to your $PATH. If you use bash, ``echo "export PATH=\$PATH:`find $(pwd)/gcc-arm-none-eabi-*-update/bin -type d`" >> ~/.bashrc`` should do what you need (you'll need to restart your terminal afterwards).
+### Fedora
 
-Alternatively, on Debian 10 and later you can directly install a sufficiently modern cross-toolchain:
+First install basics dev tools.
+
+```bash
+dnf install make automake gcc gcc-c++ kernel-devel
 ```
-apt-get install gcc-arm-none-eabi binutils-arm-none-eabi
+
+Then install required packages.
+
+```bash
+dnf install git ImageMagick libX11-devel libXext-devel freetype-devel libpng-devel libjpeg-devel pkg-config
+```
+
+Then, install GCC cross compiler for ARM.
+
+```bash
+dnf install arm-none-eabi-gcc-cs arm-none-eabi-gcc-cs-c++
+```
+
+<br>
+
+### Other Linux distro
+
+You'll need to install the latest version of GCC and make it available in your $PATH:
+
+1. Download the [GCC toolchain distributed by ARM](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads). You should obtain a `gcc-arm-none-eabi-x-linux.tar.bz2` file.
+2. Decompress that file with `tar xvfj gcc-arm-none-eabi-*-linux.tar.bz2`
+3. Add the resulting folder to your $PATH. If you use bash,
+```bash
+echo "export PATH=\$PATH:`find $(pwd)/gcc-arm-none-eabi-*-update/bin -type d`" >> ~/.bashrc
 ```
+should do what you need (you'll need to restart your terminal afterwards).
+
+Also, try to find packages listed above in your default package manager.
+
+<br>
```

Comment:
Can you maybe add a `<details>` menu to select the operating system ?

Reply:
If you want

Reply:
At this point, README or docs is redundant

Reply:
Yes, but I would think of it is more readable with collapsible menus

Reply:
The collapsable menu summary isn't displayed, and the content is displayed as raw code

Reply:
Done too.